docs/docs/changelog/index.md

```diff
@@ -1,3 +1,19 @@
+# v0.9.2
+
+# Additions
+
+- Added page caching functionality so returning to previous pages does not require an expensive page re-rendering
+- Added a help page for viewing the keybindings
+
+# Change
+
+- Updated toolchain to 1.91.1
+- src/components/page_viewer.rs now checks for a cached rendering of a page before attempting to render the page from scratch
+
+# Fixes
+
+- Fixed invalid User-Agent so the latest release can successfully hit the Wikipedia API
+
 # Unreleased v0.9.2-pre
```

Comment:
The `-pre` suffix was meant as a way of specifying changes not currently released but already in `main`. On the website you can, for example, select `v0.9.2-pre` as a version.

Since we're going to publish a new release soon, could you please merge changes for `-pre` and `0.9.2`?